docs/basics/server.md

```diff
@@ -105,8 +105,18 @@ gremlin> graph.addVertex("name", user)
 No such property: user for class: Script21
 Type ':help' or ':h' for help.
 Display stack trace? [yN]
+```
+
+Alternately, you can establish a console with a session by specifying
```

Comment:
I am not an English native speaker, but should it be 'Alternatively' rather than 'Alternately'?

Reply:
Me neither, so i decided to leave this sentence untouched even though I was confused, too.
I just looked it up and "alternately" is more suitable for the meaning of two things that are alternating.
Changed it to "alternatively" now.